buildspecs/release-to-maven.yml

```diff
@@ -14,19 +14,31 @@ phases:
   pre_build:
     commands:
     - ROOT=`pwd`
-    - CREDENTIALS=$ROOT/credentials
-    - SETTINGS_XML=$CREDENTIALS/settings.xml
-    - GPG_HOME=$CREDENTIALS/gpghome
+    - CONFIGURATION_ROOT=$ROOT/configuration
+    - SETTINGS_XML_TEMPLATE=$CONFIGURATION_ROOT/settings-template.xml
+    - SETTINGS_XML=$CONFIGURATION_ROOT/settings.xml
 
   build:
     commands:
     - RELEASE_VERSION=`mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec`
     - SONATYPE_URL="https://aws.oss.sonatype.org/service/local/repositories/releases/content/software/amazon/awssdk/aws-sdk-java/$RELEASE_VERSION/"
     - |
       if ! curl -f --head $SONATYPE_URL; then
-        mkdir -p $CREDENTIALS
-        aws s3 cp s3://aws-java-sdk-release-credentials/ $CREDENTIALS/ --recursive
-        mvn clean deploy -B -s $SETTINGS_XML -Dgpg.homedir=$GPG_HOME -Ppublishing -DperformRelease -Dspotbugs.skip -DskipTests -Dcheckstyle.skip -Djapicmp.skip -Ddoclint=none -pl !:protocol-tests,!:protocol-tests-core,!:codegen-generated-classes-test,!:sdk-benchmarks,!:module-path-tests,!:tests-coverage-reporting,!:stability-tests,!:sdk-native-image-test,!:auth-sts-testing,!:s3-benchmarks -DautoReleaseAfterClose=true -DstagingProgressTimeoutMinutes=30
+        mkdir -p $CONFIGURATION_ROOT
+
+        SDK_SIGNING_GPG_PASSPHRASE=`aws secretsmanager get-secret-value --secret-id  arn:aws:secretsmanager:us-east-1:103431983078:secret:sdk-signing-gpg-passphrase-A0H1Kq --query SecretString --output text`
```

Comment:
Nit: Maybe extract this ARN (and others) to a constant (for readability's sake and for future changes to not introduce diffs where not needed)

Reply:
+1